x/pocketcore/keeper/proof.go

```diff
@@ -50,13 +50,19 @@ func (k Keeper) SendProofTx(ctx sdk.Ctx, n client.Client, node *pc.PocketNode, p
 				ctx.Logger().Error(fmt.Sprintf("could not delete evidence is not sealed, could cause a relay leak: %s", err.Error()))
 			}
 		}
-		if evidence.NumOfProofs != claim.TotalProofs {
+
+		// In the case of evidence.NumOfProofs < claim.TotalProofs, we don't submit a proof because we cannot generate a valid merkle proof with a smaller number of relays.
```

Comment:
Can you add examples in the comment on when > and < happens?

Reply:
That's what this PR is for.

Git blame, reference back here. I rather not include such explicitness in the code for a guardrail.

Reply:
added a reference back to this PR instead